Modules/Loadable/SceneViews/Logic/vtkSlicerSceneViewsModuleLogic.cxx

```diff
@@ -453,6 +453,17 @@ void vtkSlicerSceneViewsModuleLogic::CreateSceneView(std::vector<vtkMRMLNode*> s
   sequenceBrowser->GetSynchronizedSequenceNodes(savedSequenceNodes, true);
   for (vtkMRMLSequenceNode* sequenceNode : savedSequenceNodes)
   {
+    vtkMRMLStorageNode* storageNode = sequenceNode->GetStorageNode();
+    if (storageNode)
+    {
+      // There is a storage node, but it may no longer suitable (for example, an image sequence may not contain images with different sizes,
```

Comment:
```suggestion
      // There is a storage node, but it may no longer be suitable (for example, an image sequence may now contain images with different sizes,
```